test/powershell/engine/Module/assets/testmodulerunspace/ModuleWithDependencies2/2.0/ModuleWithDependencies2.psd1

```diff
@@ -0,0 +1,113 @@
+
+@{
+
+# Script module or binary module file associated with this manifest.
+# RootModule = ''
+
+# Version number of this module.
+ModuleVersion = '2.0'
+
+# ID used to uniquely identify this module
+GUID = '0eae34da-99dd-4608-8d28-c614fe7b0841'
+
+# Author of this module
+Author = 'manikb'
+
+# Company or vendor of this module
+CompanyName = 'Unknown'
+
+# Copyright statement for this module
+Copyright = '(c) 2015 manikb. All rights reserved.'
+
+# Description of the functionality provided by this module
+Description = 'ModuleWithDependencies2 module'
+
+# Minimum version of the Windows PowerShell engine required by this module
+# PowerShellVersion = ''
+
+# Name of the Windows PowerShell host required by this module
+# PowerShellHostName = ''
+
+# Minimum version of the Windows PowerShell host required by this module
+# PowerShellHostVersion = ''
```

Comment:
i thought you mean the comments added in addition to the auto generated ones. resolved.